Doc/library/os.rst

```diff
@@ -1659,10 +1659,28 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
       Added support for pipes on Windows.
 
 
-.. function:: splice(src, dst, count, offset_src=None, offset_dst=None)
+.. function:: splice(src, dst, count, offset_src=None, offset_dst=None, flags=0)
 
    Transfer *count* bytes from file descriptor *src*, starting from offset
    *offset_src*, to file descriptor *dst*, starting from offset *offset_dst*.
+
+   *flags* argument is a bit mask that is composed by ORing
+   together zero or more of the following: :const:`SPLICE_F_MOVE`,
+   :const:`SPLICE_F_NONBLOCK`, and :const:`SPLICE_F_MORE`.
+
+   If :const:`SPLICE_F_MOVE` is specified it hints the kernel
+   to move pages instead of copying, but pages may still be copied if the
+   kernel cannot move the pages from the pipe.
+
+   If :const:`SPLICE_F_NONBLOCK` is specified it hints the kernel
+   to not block on I/O.  This makes the splice pipe
+   operations nonblocking, but splice may nevertheless
+   block because the file descriptors that are spliced
+   to/from may block.
+
+   If :const:`SPLICE_F_MORE` is specified it hints the kernel
+   that more data will be coming in a subsequent splice.
```

Comment:
```suggestion
   * If :const:`SPLICE_F_MOVE` is specified,
     the kernel is asked to move pages instead of copying,
     but pages may still be copied if the kernel cannot move the pages from the pipe.

   * If :const:`SPLICE_F_NONBLOCK` is specified,
     the kernel is asked to not block on I/O.
     This makes the splice pipe operations nonblocking,
     but splice may nevertheless block because the spliced file descriptors may block.

   * If :const:`SPLICE_F_MORE` is specified,
     it hints to the kernel that more data will be coming in a subsequent splice.
```